spec/lib/backup_restore/backuper_spec.rb

```diff
@@ -16,4 +16,13 @@
 
     expect(backuper.send(:get_parameterized_title)).to eq("coding-horror")
   end
+
+  describe '#notify_user' do
+    it 'include upload' do
+      backuper = BackupRestore::Backuper.new(Discourse.system_user.id)
+      expect { backuper.send(:notify_user) }
+        .to change { Topic.private_messages.count }.by(1)
+        .and change { Upload.count }.by(1)
```

Comment:
I think it is better to be more specific here where we test the name of the upload and maybe the file type?

Reply:
That is a good idea.